docs/book/introduction.md

```diff
@@ -0,0 +1,25 @@
+# Introduction
+
+Giving feedback to a user is an important part of a good application.
+Flash messages are used to notify the user about a successful form submit, failure on saving in the database, wrong authentification credentials or something similar.
+At the end of a request, a message is created and shown to the user at the next request.
```

Comment:
This sentence is a bit garbled up - confusing wording makes it unclear whether the rendering is happening in one request or the other

Reply:
How about "A message is created at the end of one request, and shown to the user in the next request."?